src/compile/data/aggregate.ts

```diff
@@ -11,11 +12,9 @@ function addDimension(dims: {[field: string]: boolean}, fieldDef: FieldDef) {
     dims[field(fieldDef, {binSuffix: 'start'})] = true;
     dims[field(fieldDef, {binSuffix: 'end'})] = true;
 
-    // const scale = model.scale(channel);
-    // if (scaleType(scale, fieldDef, channel, model.mark()) === ScaleType.ORDINAL) {
-    // also produce bin_range if the binned field use ordinal scale
-    dims[field(fieldDef, {binSuffix: 'range'})] = true;
-    // }
+    if (fieldDef.type === ORDINAL) {
```

Comment:
Ok -- this fix is correct because it's in bin so it can only be `bin-ordinal`. 

However, `hasDiscreteDomain` does not always imply `type === ORDINAL`.